.pylintrc

```diff
@@ -0,0 +1,19 @@
+[tool.pylint.main]
+load-plugins = pylint.extensions.docparams
+disable = R0913, R0914, W3101, R1720, R1705
+
+# R0913: too-many-arguments
+# R0914: too-many-locals
+# W3101: missing-timeout
```

Comment:
We can add this in, I wasn't sure if we were to add it how many seconds would be appropriate.

Reply:
Neither am I, especially since some of the requests are very large. Maybe this would be too troublesome to add.

Reply:
Shall we just keep it in the ignore list for now then?

> Neither am I, especially since some of the requests are very large. Maybe this would be too troublesome to add.

Reply:
Lets do that.
Resolved